packages/api-docs-builder-core/tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "strict": true,
     "baseUrl": "./",
     "paths": {
-      "react-docgen": ["../api-docs-builder/react-docgen.d.ts"]
```

Comment:
Why is this no longe required?

Reply:
React-docgen types live in a separate package and are simply added as a dev dependency.